ui/components/app/hold-to-reveal-button/hold-to-reveal-button.js

```diff
@@ -93,6 +92,7 @@ export default function HoldToRevealButton({ buttonText, onLongPressed }) {
         <Box className="hold-to-reveal-button__absolute-fill">
           <svg className="hold-to-reveal-button__circle-svg">
             <circle
+              aria-label="hold-to-reveal-circle"
```

Comment:
Are these for folks that use screen readers and if so can you explain to me how they are intended to work? I thought aria-labels should be written as if you were reading it not kebab-case? I maybe if we had locked and unlocked?

Reply:
i'm using them for testing than for screen readers. i can make the change to locked and unlocked